Directory.Packages.props

```diff
@@ -93,19 +93,19 @@
     <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="$(MicrosoftExtensionsHttpResiliencePackageVersion)" />
     <PackageVersion Include="Microsoft.Extensions.TimeProvider.Testing" Version="$(MicrosoftExtensionsTimeProviderTestingVersion)" />
     <!-- external dependencies -->
-    <PackageVersion Include="Confluent.Kafka" Version="2.5.2" />
+    <PackageVersion Include="Confluent.Kafka" Version="2.5.3" />
     <PackageVersion Include="Dapper" Version="2.1.44" />
     <PackageVersion Include="DnsClient" Version="1.8.0" />
-    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.15.2" />
-    <PackageVersion Include="Google.Protobuf" Version="3.27.3" />
+    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.15.6" />
+    <PackageVersion Include="Google.Protobuf" Version="3.28.0" />
     <PackageVersion Include="Grpc.AspNetCore" Version="2.65.0" />
     <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.65.0" />
-    <PackageVersion Include="Grpc.Tools" Version="2.65.0" />
+    <PackageVersion Include="Grpc.Tools" Version="2.66.0" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="KubernetesClient" Version="14.0.9" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.2" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.9.3" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.9.3" />
+    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.10.0" />
```

Comment:
Ping some devs from the dashboard to check we want this? UI can be picky.

Reply:
@adamint @JamesNK can you confirm this is OK?

Reply:
It is ok or not based on whether anything in the dashboard is broken. The only way to check is go through the dashboard UI and check to see if anything is broken.

Reply:
What am I looking for? Would it be better to just revert this and have someone in the dashboard side to update and validate instead?

Reply:
You look at everything. But I created a PR here for it: https://github.com/dotnet/aspire/pull/5670

There are regressions.

Reply:
Yup, two tests were failing, I've reverted those changes from this PR to unblock it.